paperpress.js

```diff
@@ -114,13 +115,33 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var attr = contentData.attributes
+
+		item = {
+			title: attr.title,
+			slug: attr.slug,
```

Comment:
can we add fallback in case `title` or `slug` aren't defined in the frontMatter attributes?
```
title: attr.title || name,
slug: attr.slug || slug,
```